lib/youtube-dl.js

```diff
@@ -238,6 +238,10 @@ ytdl.exec = function exec(url, args, options, callback) {
  * @returns {Object}
  */
 function parseInfo(data) {
+  // Youtube-dl might return just an url as a string when using the "-g" or "--get-url" flag
```

Comment:
Little typo: `Youtube-dl` → `youtube-dl`

Reply:
Done in e968072